src/discussions/utils.js

```diff
@@ -1,15 +1,23 @@
 /* eslint-disable import/prefer-default-export */
+import { useContext } from 'react';
+
 import { getIn } from 'formik';
 import { useRouteMatch } from 'react-router';
 
-import { Routes } from '../data/constants';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  Delete, Edit, Flag, Pin, QuestionAnswer, VerifiedBadge,
+} from '@edx/paragon/icons';
+
+import { EditActions, Routes } from '../data/constants';
+import messages from './messages';
 
-export function buildIntlSelectionList(options, intl, messages) {
+export function buildIntlSelectionList(options, intl, messagesData) {
```

Comment:
Renamed because we are not importing messages, so there is already a variable called messages here.